server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -3684,8 +3683,13 @@ public void testAbortedSnapshotDuringInitDoesNotStart() throws Exception {
 
             // The deletion must set the snapshot in the ABORTED state
             assertBusy(() -> {
-                SnapshotsStatusResponse status = client.admin().cluster().prepareSnapshotStatus("repository").setSnapshots("snap").get();
-                assertThat(status.getSnapshots().iterator().next().getState(), equalTo(State.ABORTED));
+                try {
+                    SnapshotsStatusResponse status =
+                        client.admin().cluster().prepareSnapshotStatus("repository").setSnapshots("snap").get();
+                    assertThat(status.getSnapshots().iterator().next().getState(), equalTo(State.ABORTED));
+                } catch (Exception e) {
+                    throw new AssertionError(e);
```

Comment:
maybe add a comment to say that here we rethrow in AssertionError to force assertBusy to retry.

Reply:
Sure, I added it :)

Reply:
@DaveCTurner thanks! I think I have an idea where this is coming from still ... Will fix tomorrow :)